ci/jenkins/mellanox/jobs/00-functions.yaml

```diff
@@ -0,0 +1,133 @@
+- builder:
+    name: run-ci
+    builders:
+         - shell:
```

Comment:
nit: same indention length in same yaml file is appreciated. I prefer `builder-pending-label` one.

Reply:
Done